crates/bevy_ecs/src/bundle.rs

```diff
@@ -928,3 +928,28 @@ fn initialize_dynamic_bundle(
 
     (id, storage_types)
 }
+
+#[cfg(test)]
+mod tests {
+    use bevy_ecs_macros::{Bundle, Component};
+
+    use crate as bevy_ecs;
+
+    #[derive(Component)]
+    struct ComponentA(u32);
+
+    #[derive(Component)]
+    struct ComponentB(u32);
+
+    #[derive(Bundle)]
+    struct Simple(ComponentA);
+
+    #[derive(Bundle)]
+    struct Tuple(Simple, ComponentB);
+
+    #[derive(Bundle)]
+    struct Record {
+        field0: Simple,
+        field1: ComponentB,
+    }
+}
```

Comment:
Could you move the `fn bundle_derive` test in `crates/bevy_ecs/src/lib.rs` at line 149 to this module? Or the reverse? I think it makes sense to put related tests together.

Reply:
moved them in `crates/bevy_ecs/src/lib.rs` since all other tests also there